great-idea/css/index.css

```diff
@@ -63,4 +63,145 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+	.navigation{
+		display: flex;
+		justify-content: space-around;
+		align-items: center;
+	
+	}
+	.navigation a {
+		padding: 30px;
+		border: 0;
+		font-size: 100%;
+		color: grey;
+		text-decoration: none;
+		list-style-type: none;
+	}
+	.container{
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-content: center;
+		padding: 75px;
+	}
+
+	h1{
+		font-size: 60px;
+	}
+
+	.head{
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+	
+	}
+	
+	.innovation{
+		width: 49%;
+		height: 49%;
+		text-align: center;
+	}
+	.button{
+		align-self: center;
+		width: 165px;
+		height: 40px;
+		font-size: 20px;
+		padding: 5px;
+		border: 1px solid black;
+		text-align: center;
+		order: 1;
+		margin: 0px 0px 50px 200px;
+		
+		
+		
+
+	}
+	
+	.topContent{
+		border-top: 1px solid black;
+		display: flex;
+		justify-content: space-evenly;
+	
+	}
+
+	.features{
+		width: 50%;
+		padding: 30px 0px 30px 0px;
+		margin: 10px;
+	}
+
+	.about{
+		width: 49%;
+		padding: 30px 0px 30px 0px;
+		margin: 10px;
+	}
+
+	.bottomContent{
+		border-bottom: 1px solid black;
+		display: flex;
+		justify-content: space-evenly;
+	}
+	
+	.item{
+		padding: 30px 0px 30px 0px;
+	}
+
+	.contact{
+		line-height: 1.6;
+		padding: 30px 0px 30px 0px;
+
+	}
+
+	footer{
+		text-align: center;
+	}
+
+	.firstRow{
+		display: flex;
+		margin: 0;
+		align-content: space-around;
+		
+	}
+
+	.box{
+		border: 1px solid black;
+		background-color: #F4F3F4;
+		padding: 25px;
+		margin: 25px 10px 0px 10px;
+	}
+
+	.button2{
+		border: 1px solid black;
+		border-radius: 5px;
+		background-color: white;
+		height: 30px; 
+		width: 150px;
+		padding: 5px;
+		text-align: center;
+		margin: 10px  0px 0px 0px;
+	}
+
+	.bottom{
+		padding: 25px;
+		margin: 25px 10px 0px 10px;
+		display: flex;
+		flex-wrap:wrap;
+	}
+.bottomPage{
+		padding: 25px;
+		margin: 25px 10px 0px 10px;
+		border-top: 1px solid black;
+}
+
+	ul {
+		list-style-type: disc;
+		padding: 15px;
+		line-height: 1.6;
+		
+	}
+
+	.services-info-img{
+		display: flex;
```

Comment:
Adding `display: flex` here isn't doing anything. Display:flex will only help if you have multiple elements in a container that you are adding flex to